sources/fsizes.h

```diff
@@ -126,10 +126,10 @@
 #define MAXFPATCHES 256
 #define SORTIOSIZE 200000L
 
-#define SSMALLBUFFER 500000L
-#define SSMALLOVERFLOW 800000L
+#define SSMALLBUFFER 2560016L
+#define SSMALLOVERFLOW 3840032L
 #define STERMSSMALL 10000L
-#define SLARGEBUFFER 4000000L
+#define SLARGEBUFFER 26880512L
 #define SMAXPATCHES 64
```

Comment:
Can we round these values to make them a bit cleaner?

Reply:
I used the current values since they are not actually a change in the sizes; FORM has effectively been using them all along.

We could use, eg, `2560000L, 3840000L`. `26880000L` for SLARGEBUFFER gets set back up since it is set to `MaxFpatches*((IObuffersize+COMPINC)*sizeof(WORD)+2*AM.MaxTer)  - SmallEsize`.

Changing the default buffer sizes potentially has implications on some of the tests, though the ones that are sensitive to the buffer sizes should be setting them in the test script (and usually the test cases need much smaller-than-default values for a quickly-running test).

Reply:
OK, then we'll leave these values as they are.